tests/models/test_record.py

```diff
@@ -69,6 +69,35 @@ def test_sets_priority(self):
         self.assertEqual(record.priority, 10)
 
 
+class TestCreateSRVRecord(FakedServiceTestCase):
+    def setUp(self):
+        self.user = User.get(email='foo@example.net')
+        self.zone = self.user.get_zone_by_name('example.com')
+        self.test_record = self.zone.create_record(
+            'bar.example.com', 'SRV', '',
```

Comment:
Yeah, I wasn't entirely sure what to do there as content doesn't seem to be used for SRV records and I didn't want to put in anything misleading.